experimental/go-sdk/bindings/core/generate.go

```diff
@@ -6,6 +6,6 @@ import (
 
 // generate core binary and embed into this package
 //
-//go:generate go build -C ../.. -o lib/core/embed-core.bin cmd/core/main.go
+//go:generate ./../../scripts/build_embed.sh bindings/core/embed-core.bin
```

Comment:
the `../../` seems like a sign that `build_embed.sh` should be in this directory, especially if it's not used elsewhere

Reply:
it used in the example here as well: https://github.com/wandb/wandb/pull/9140/files#diff-abe6c02ef6d86125bc7622e4f603be569b50b757eefe802dba22f87962f7e917R11